diff --git a/IEdgeInsights/EtcdUI/Dockerfile b/IEdgeInsights/EtcdUI/Dockerfile
index d7fb47e..5bd71c0 100644
--- a/IEdgeInsights/EtcdUI/Dockerfile
+++ b/IEdgeInsights/EtcdUI/Dockerfile
@@ -51,7 +51,8 @@ COPY --from=common /usr/local/lib/python3.6/dist-packages/ /usr/local/lib/python
 
 
 COPY etcdkeeper ./etcdkeeper/
-
+ENV GO111MODULE on
+ENV GOPROXY "https://goproxy.cn"
 RUN cd ./etcdkeeper/src/etcdkeeper \
     && go build -o etcdkeeper main.go \
     && mv etcdkeeper ../../
diff --git a/IEdgeInsights/Grafana/Dockerfile b/IEdgeInsights/Grafana/Dockerfile
index c640f10..7cf74ea 100644
--- a/IEdgeInsights/Grafana/Dockerfile
+++ b/IEdgeInsights/Grafana/Dockerfile
@@ -29,7 +29,7 @@ LABEL description="Grafana image"
 ENV PYTHONPATH ${PYTHONPATH}:./..
 
 ARG GRAFANA_VERSION
-
+RUN sed -i 's/archive.ubuntu.com/mirrors.aliyun.com/g' /etc/apt/sources.list
 RUN apt-get update && \
     apt-get -y --no-install-recommends install libfontconfig curl ca-certificates && \
     apt-get clean && \
diff --git a/IEdgeInsights/InfluxDBConnector/Dockerfile b/IEdgeInsights/InfluxDBConnector/Dockerfile
index ae938f7..4e12e3f 100644
--- a/IEdgeInsights/InfluxDBConnector/Dockerfile
+++ b/IEdgeInsights/InfluxDBConnector/Dockerfile
@@ -29,7 +29,8 @@ WORKDIR ${GO_WORK_DIR}
 
 ENV INFLUXDB_GO_PATH ${GOPATH}/src/github.com/influxdata/influxdb
 RUN mkdir -p ${INFLUXDB_GO_PATH} && \
-    git clone https://github.com/influxdata/influxdb ${INFLUXDB_GO_PATH} && \
+    # git clone https://github.com/influxdata/influxdb ${INFLUXDB_GO_PATH} && \
+    git clone https://gitee.com/mirrors/InfluxDB.git ${INFLUXDB_GO_PATH} && \
     cd ${INFLUXDB_GO_PATH} && \
     git checkout -b v1.6.0 tags/v1.6.0
 
diff --git a/IEdgeInsights/Kapacitor/Dockerfile b/IEdgeInsights/Kapacitor/Dockerfile
index 0805ddf..1f9b25b 100644
--- a/IEdgeInsights/Kapacitor/Dockerfile
+++ b/IEdgeInsights/Kapacitor/Dockerfile
@@ -61,7 +61,8 @@ RUN conda config --add channels intel\
     && mv ${PY_WORK_DIR}/miniconda/lib/python3.7/ /usr/local/lib/
 
 # Installing EIS related libs
-RUN git clone https://github.com/kragniz/python-etcd3 && \
+#RUN git clone https://github.com/kragniz/python-etcd3 && \
+RUN git clone https://gitee.com/mirrors/python-etcd3.git && \
     cd python-etcd3 && \
     git checkout -b ${PY_ETCD3_VERSION} ${PY_ETCD3_VERSION} && \
     python3.7 setup.py install && \
diff --git a/IEdgeInsights/common/dockerfiles/Dockerfile.eisbase b/IEdgeInsights/common/dockerfiles/Dockerfile.eisbase
index 3b533ed..7df9026 100644
--- a/IEdgeInsights/common/dockerfiles/Dockerfile.eisbase
+++ b/IEdgeInsights/common/dockerfiles/Dockerfile.eisbase
@@ -34,9 +34,12 @@ WORKDIR ${GO_WORK_DIR}
 
 #Installing Go and dep package manager tool for Go
 ARG GO_VERSION
+RUN sed -i 's/archive.ubuntu.com/mirrors.aliyun.com/g' /etc/apt/sources.list
+
 RUN apt-get update && \
     apt-get install -y wget git build-essential pkg-config iputils-ping g++ && \
-    wget https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz && \
+    # wget https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz && \
+    wget https://golang.google.cn/dl/go${GO_VERSION}.linux-amd64.tar.gz && \
     tar -C /usr/local -xzf go${GO_VERSION}.linux-amd64.tar.gz
 
 ARG DEBIAN_FRONTEND=noninteractive
@@ -85,7 +88,8 @@ ENV CGO_LDFLAGS "$CGO_LDFLAGS -z noexecstack -z relro -z now"
 ENV ETCD_GO_PATH ${GOPATH}/src/go.etcd.io/etcd
 ENV ETCD_GO_VER 0c787e26bcd102c3bb14050ac893b07ba9ea029f
 RUN mkdir -p ${ETCD_GO_PATH} && \
-    git clone https://github.com/etcd-io/etcd ${ETCD_GO_PATH} && \
+    # git clone https://github.com/etcd-io/etcd ${ETCD_GO_PATH} && \
+    git clone https://gitee.com/mirrors/etcd.git ${ETCD_GO_PATH} && \
     cd ${ETCD_GO_PATH} && \
     git checkout -b ${ETCD_GO_VER} ${ETCD_GO_VER}
 
@@ -105,12 +109,15 @@ RUN ln -sf /usr/bin/pip3 /usr/bin/pip3.6
 
 # Upgrading pip version
 RUN pip3 install --upgrade pip
+RUN mkdir -p ~/.pip && \
+    printf '[global]\n\ttrusted-host=mirrors.aliyun.com\n\tindex-url=http://mirrors.aliyun.com/pypi/simple' > ~/.pip/pip.conf
 
 # Installing python JSON schema validator
 RUN pip3 install jsonschema==3.2.0
 
 ENV PY_ETCD3_VERSION cdc4c48bde88a795230a02aa574df84ed9ccfa52
-RUN git clone https://github.com/kragniz/python-etcd3 && \
+# RUN git clone https://github.com/kragniz/python-etcd3 && \
+RUN git clone https://gitee.com/mirrors/python-etcd3.git && \
     cd python-etcd3 && \
     git checkout -b ${PY_ETCD3_VERSION} ${PY_ETCD3_VERSION} && \
     python3.6 setup.py install && \
diff --git a/IEdgeInsights/common/eis_libs_installer.sh b/IEdgeInsights/common/eis_libs_installer.sh
index d98dbb8..15010cd 100644
--- a/IEdgeInsights/common/eis_libs_installer.sh
+++ b/IEdgeInsights/common/eis_libs_installer.sh
@@ -35,7 +35,8 @@ install_go_dependencies () {
 install_go () {
     apt-get update && \
     apt-get install -y wget git build-essential pkg-config iputils-ping g++ && \
-    wget https://dl.google.com/go/go1.12.linux-amd64.tar.gz && \
+    # wget https://dl.google.com/go/go1.12.linux-amd64.tar.gz && \
+    wget https://golang.google.cn/dl/go1.12.linux-amd64.tar.gz && \
     tar -C /usr/local -xzf go1.12.linux-amd64.tar.gz
     export GOPATH=/home/$USER/go/
     mkdir -p $GOPATH/src
@@ -67,7 +68,8 @@ wget -O- https://cmake.org/files/v3.15/cmake-3.15.0-Linux-x86_64.tar.gz | \
 apt-get install -y python3-distutils
 
 export PY_ETCD3_VERSION=cdc4c48bde88a795230a02aa574df84ed9ccfa52 && \
-    git clone https://github.com/kragniz/python-etcd3 && \
+    # git clone https://github.com/kragniz/python-etcd3 && \
+    git clone https://gitee.com/mirrors/python-etcd3.git && \
     cd python-etcd3 && \
     git checkout $PY_ETCD3_VERSION && \
     python3.6 setup.py install && \
